internal/provider/resources/flow.go

```diff
@@ -0,0 +1,355 @@
+			},
+			"account_id": schema.StringAttribute{
+				CustomType:  customtypes.UUIDType{},
+				Description: "Account ID (UUID), defaults to the account set in the provider",
+				Optional:    true,
+			},
+			"workspace_id": schema.StringAttribute{
+				Optional:    true,
+				CustomType:  customtypes.UUIDType{},
+				Description: "Workspace ID (UUID)",
+			},
+			"name": schema.StringAttribute{
+				Description: "Name of the flow",
+				Required:    true,
+			},
+
+			"tags": schema.ListAttribute{
+				Description: "Tags associated with the flow",
+				ElementType: types.StringType,
+				Optional:    true,
+				Computed:    true,
+				Default:     listdefault.StaticValue(defaultEmptyTagList),
+				PlanModifiers: []planmodifier.List{
+					listplanmodifier.RequiresReplace(),
+				},
+			},
+		},
+	}
+}
+
+// copyFlowToModel copies an api.Flow to a FlowResourceModel.
+func copyFlowToModel(ctx context.Context, flow *api.Flow, model *FlowResourceModel) diag.Diagnostics {
+	model.ID = types.StringValue(flow.ID.String())
+	model.Created = customtypes.NewTimestampPointerValue(flow.Created)
+	model.Updated = customtypes.NewTimestampPointerValue(flow.Updated)
+	model.Name = types.StringValue(flow.Name)
+
+	tags, diags := types.ListValueFrom(ctx, types.StringType, flow.Tags)
+	if diags.HasError() {
+		return diags
+	}
+	model.Tags = tags
+
+	return nil
+}
+
+// Create creates the resource and sets the initial Terraform state.
+func (r *FlowResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	var model FlowResourceModel
+
+	// Populate the model from resource configuration and emit diagnostics on error
+	resp.Diagnostics.Append(req.Config.Get(ctx, &model)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	var tags []string
+	resp.Diagnostics.Append(model.Tags.ElementsAs(ctx, &tags, false)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	client, err := r.client.Flows(model.AccountID.ValueUUID(), model.WorkspaceID.ValueUUID())
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error creating flows client",
+			fmt.Sprintf("Could not create flows client, unexpected error: %s. This is a bug in the provider, please report this to the maintainers.", err.Error()),
+		)
+	}
+
+	flow, err := client.Create(ctx, api.FlowCreate{
+		Name: model.Name.ValueString(),
+		Tags: tags,
+	})
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error creating flow",
+			fmt.Sprintf("Could not create flow, unexpected error: %s", err),
+		)
+
+		return
+	}
+
+	resp.Diagnostics.Append(copyFlowToModel(ctx, flow, &model)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	resp.Diagnostics.Append(resp.State.Set(ctx, &model)...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+}
+
+// Read refreshes the Terraform state with the latest data.
+func (r *FlowResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+	var model FlowResourceModel
```

Comment:
Let's change this variable name to `state` per my earlier comment.

Reply:
done